tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -103,6 +104,67 @@ where
 		Ok(json!(json_value.to_hex()))
 	});
 
+	// author_getEnclaveAccountId
+	let state_account_id = state.clone();
+	let author_get_enclave_account_id: &str = "author_getEnclaveAccountId";
```

Comment:
You can actually get it via parachain storage.

If you really need it, it can be derived from the shielding key pair I think:
https://github.com/litentry/litentry-parachain/blob/55c2e1642408e8475164dd665a9cbd219c2a5bc0/tee-worker/core-primitives/stf-state-handler/src/state_initializer.rs#L61